.openshift-ci/ci_tests.py

```diff
@@ -75,6 +84,14 @@ def run(self):
             OperatorE2eTest.E2E_TEST_TIMEOUT_SEC,
         )
 
+        print("Storing test-e2e-deployed results in JUnit format")
+        self.run_with_graceful_kill(
```

Comment:
Would be nice to refactor this `print`+`run` into a couple of methods:
- one in `OperatorE2eTest` which contains pieces specific to kuttl artifacts, or perhaps specific to operator tests; it would delegate to:
- another generic method in `BaseTest` that calls `store-artifacts.sh` and would potentially be useful to other subclasses.